src/RevMem.cc

```diff
@@ -877,6 +832,17 @@ uint64_t RevMem::ShrinkMemSeg(std::shared_ptr<MemSegment> Seg, const uint64_t Ne
   }
 }
 
+/// @brief This function is called from the loader to initialize the heap
+/// @param EndOfStaticData: The address of the end of the static data section (ie. end of .bss section)
+void RevMem::InitHeap(const uint64_t& EndOfStaticData){
+  if( EndOfStaticData <= 0 ){
+    output->fatal(CALL_INFO, 7, "EndOfStaticData = 0x%lx which is less than or equal to 0. This is a bug.", EndOfStaticData);
```

Comment:
Is this reachable since EndOfStaticData is unsigned? Or is there a risk the compiler may cast it as signed (given "0" is ambiguous) which would give strange results

Reply:
Theoretically reachable if `EndofStaticData == 0`; but not otherwise..... I would suggest revising to: 
```
if( EndOfStaticData <= 0x00ull )
```

Reply:
Fixed in most recent push